src/lib/rest/rest.cpp

```diff
@@ -1068,6 +1071,8 @@ void restInit
   rushHost      = _rushHost;
   rushPort      = _rushPort;
 
+  strcpy(restAllowedOrigin, _allowedOrigin);
```

Comment:
Use strncpy instead. The style guide script will complain ...
Wish we had the style guide check running ...